tools/unicode_properties_parse/download_unicode_data_files.py

```diff
@@ -0,0 +1,20 @@
+from urllib.request import urlretrieve
+
+
+Unicode_data_files = {
+    "DerivedCoreProperties.txt": "https://www.unicode.org/Public/UCD/latest/ucd/DerivedCoreProperties.txt",
+    "DerivedGeneralCategory.txt": "https://www.unicode.org/Public/UCD/latest/ucd/extracted/DerivedGeneralCategory.txt",
+    "EastAsianWidth.txt": "https://www.unicode.org/Public/UCD/latest/ucd/EastAsianWidth.txt",
+    "GraphemeBreakProperty.txt": "https://www.unicode.org/Public/UCD/latest/ucd/auxiliary/GraphemeBreakProperty.txt",
+    "GraphemeBreakText.txt": "https://www.unicode.org/Public/UCD/latest/ucd/auxiliary/GraphemeBreakTest.txt",

```

Comment:
Heh, there's a pre-existing typo in our `.gitignore` which we should fix now.

Reply:
`.gitignore` is correct. The typo is in `grapheme_break_test_data_gen.py`, which I erroneously copied over.

https://github.com/microsoft/STL/blob/8b081e26ba016970ce3338cb483ff10f2ade30a5/tools/unicode_properties_parse/grapheme_break_test_data_gen.py#L94

Reply:
Ha, thanks for noticing the mistake in my mistake correction! :repeat: :joy_cat: